content/en/docs/v3/about/maturity-matrix.md

```diff
@@ -5,14 +5,18 @@ description: Jenkins X 3.x features maturity assessment on different cloud provi
 weight: 170
 ---
 
+<script src="node_modules/medium-zoom/dist/medium-zoom.min.js"></script>
+
 This maturity matrix descibes the status of the Jenkins X 3.x __Alpha__ release.  This will continue to evolve as we progress towards Beta.
 
-<img src="/images/v3/jx-v3alpha-maturity-matrix.png">
+<img class="data-zoomable" src="/images/v3/jx-v3alpha-maturity-matrix.png">
 
-<img src="/images/v3/jx-v3alpha-color-rep.png">
+<img class="data-zoomable" src="/images/v3/jx-v3alpha-color-rep.png">
 
 <br />
 
 This maturity matrix describes the tentative plan post Jenkins X v3.0 __GA__ release. This matrix will continue to evolve as we progress further.
 
-<img src="/images/v3/jx-v3ga-maturity-matrix.png">
+<img class="data-zoomable" src="/images/v3/jx-v3ga-maturity-matrix.png">
+
+<script src="static/apidocs/zoom.js"></script>
```

Comment:
Why `static/apidocs/zoom.js`? I would prefer `js/zoom.js` (You don't need static folder name in the src part of the script)

Reply:
@ankitm123 i tried to make another html file but its not supporting or i should put this in an existing html file. Because i didn't find any relatable html file with maturity-matrix where shortcodes should be written

Reply:
> @ankitm123 i tried to make another html file but its not supporting or i should put this in an existing html file. Because i didn't find any relatable html file with maturity-matrix where shortcodes should be written

Also @ankitm123 can you please help me with this problem i asked